modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc

```diff
@@ -7,13 +7,21 @@
 [id="defining-the-list-of-images-to-pull"]
 = Defining the list of images to pull
 
-The {image-puller-name-short} can pre-pull most images, including scratch images such as `che-machine-exec`. However, images that mount volumes in the Dockerfile, such as `traefik`, are not supported for pre-pulling on OpenShift 3.11.
+The {image-puller-name-short} can pre-pull most images, including scratch images. 
 
 .Procedure
 
-. Gather a list of relevant container images to pull by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt` URL.
+. Get the latest {devworkspace} and {prod-short} operator bundle images, and use `podman` and `yq` to parse the related images from the ClusterServiceVersion YAML file in the /manifests/ folder to a simple list of container images:
 
-. Determine images from the list for pre-pulling. For faster workspace startup times, consider pulling workspace related images such as `che-theia`, `che-machine-exec`, `che-theia-endpoint-runtime-binary`, and plug-in sidecar images.
+```
+cd /tmp && \
+curl -sSLkO https://raw.githubusercontent.com/redhat-developer/devspaces/devspaces-3-rhel-8/product/getRelatedImages.sh && \
+chmod +x getRelatedImages.sh && \
+./getRelatedImages.sh -t {prod-ver-patch} -d {devworkspace-operator-version} --quiet
+
+```
+
+. For faster workspace startup times, consider pre-pulling workspace related images such as `che-code`, `che-theia`, `che-machine-exec`, `che-theia-endpoint-runtime-binary`, and the Universal Developer Image (`udi`) images.
```

Comment:
```suggestion
TIP: For faster workspace startup times, consider pre-pulling workspace related images such as `che-code`, `che-theia`, `che-machine-exec`, `che-theia-endpoint-runtime-binary`, and the Universal Developer Image (`udi`) images.
```

Reply:
that's not a tip, that's the whole point of this document :D

Reply:
It's interesting that it's at the very end, let me rework that.